src/components/Card.js

```diff
@@ -27,44 +29,17 @@ const cardStyle = makeStyles(theme => ({
   }
 }));
 
-const Card = ({ content, dataPolicy, protocols, sampleUnits, histogram }) => {
-  const classes = cardStyle();
-  const { type, body, legend, title, hardCoralCovers, reefFishBiomass, header } = content;
-  const privatePolicyCheck = type === 'pieChart' && dataPolicy === 'private';
-
-  const loaderType = type === 'text' ? <TextLoader /> : <ChartLoader />;
-
-  const subAttributeItem = hardCoralCovers ? (
-    <Typography m={1}>Hard coral cover: {hardCoralCovers} %</Typography>
-  ) : (
-    <Typography m={1}>Reef fish biomass: {reefFishBiomass} kg/ha</Typography>
-  );
-
-  const subItems = type === 'pieChart' && (
-    <Box>
-      <Typography m={1}>Data sharing: {dataPolicy}</Typography>
-      <Typography m={1}>Sample units: {sampleUnits} </Typography>
-      {subAttributeItem}
-    </Box>
-  );
-
-  const subTitle = header && <Typography variant="h6">{header}</Typography>;
-
-  // eslint-disable-next-line
-  const downLoadButton = type === 'pieChart' && (
-    <ThemeProvider theme={theme.cardButton}>
-      <ButtonStyle notAllowed={privatePolicyCheck} boxShadow={true}>
-        <Box p={1} display="flex" justifyContent="center">
-          <DownloadIcon fontSize="small" className={classes.iconProperty} />
-          <Typography variant="body1" display="inline">
-            Download Data
-          </Typography>
-        </Box>
-      </ButtonStyle>
-    </ThemeProvider>
-  );
-
-  const contentType =
+const CardContent = ({
```

Comment:
Because of the amount that this component takes in, this component should live in its own file. I suggest renaming to whatever you rename so that it follows whatever you rename Card to (see suggestion below)